src/drops/utils/PublicMulticall.sol

```diff
@@ -7,7 +7,7 @@ abstract contract PublicMulticall {
   /**
    * @dev Receives and executes a batch of function calls on this contract.
    */
-  function multicall(bytes[] calldata data) public virtual returns (bytes[] memory results) {
+  function multicall(bytes[] memory data) public virtual returns (bytes[] memory results) {
```

Comment:
why needed to be updated from calldata to memory? data needs to be manipulated now?